src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml

```diff
@@ -68,42 +101,81 @@ limitations under the License.
     <define>TIME_SYNCHRONIZATION_CLUSTER</define>
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
-    <description>Accurate time is required for a number of reasons, including scheduling, display and validating
-security materials.</description>
+    <description>Accurate time is required for a number of reasons, including scheduling, display and validating security materials.</description>
     <!-- Base data types -->
     <attribute side="server" code="0x0000" define="UTC_TIME" type="epoch_us" isNullable="true" optional="false">UTCTime</attribute>
     <attribute side="server" code="0x0001" define="GRANULARITY" type="GranularityEnum" default="0x00" optional="false">Granularity</attribute>
     <attribute side="server" code="0x0002" define="TIME_SOURCE" type="TimeSourceEnum" default="0x00" optional="true">TimeSource</attribute>
-    <attribute side="server" code="0x0003" define="TRUSTED_TIME_NODE_ID" type="node_id" writable="true" isNullable="true" optional="false">
-	<description>TrustedTimeNodeId</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="administer"/>
-    </attribute>		
-    <attribute side="server" code="0x0004" define="DEFAULT_NTP" type="CHAR_STRING" length="128" writable="true" isNullable="true" optional="true">
-	<description>DefaultNtp</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="administer"/>
-    </attribute>		
-    <attribute side="server" code="0x0005" define="TIME_ZONE" type="ARRAY" entryType="TimeZoneStruct" min="1" max="2" writable="true" optional="true">
-	<description>TimeZone</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="manage"/>
-    </attribute>
-    <attribute side="server" code="0x0006" define="DST_OFFSET" type="ARRAY" entryType="DSTOffsetStruct" max="20" writable="true" optional="true">
-	<description>DSTOffset</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="manage"/>
-    </attribute>		
+    <attribute side="server" code="0x0003" define="TRUSTED_TIME_SOURCE" type="TrustedTimeSourceStruct" isNullable="true" optional="true">TrustedTimeSource</attribute>
+    <attribute side="server" code="0x0004" define="DEFAULT_NTP" type="CHAR_STRING" length="128" isNullable="true" optional="true">DefaultNTP</attribute>
+    <attribute side="server" code="0x0005" define="TIME_ZONE" type="ARRAY" entryType="TimeZoneStruct" optional="true">TimeZone</attribute>
+    <attribute side="server" code="0x0006" define="DST_OFFSET" type="ARRAY" entryType="DSTOffsetStruct" optional="true">DSTOffset</attribute>
     <attribute side="server" code="0x0007" define="LOCAL_TIME" type="epoch_us" default="0x00" isNullable="true" optional="true">LocalTime</attribute>
```

Comment:
The spec states default is null and here it is set to default="0x00"? Or this this two separate things?

Reply:
It's not two separate things....

But there's no way to represent "default is null" in the ZAP XML.  See https://github.com/project-chip/zap/issues/608

Reply:
I see, but even in that case I guess 0 as the default is not correct?